api/src/services/events.js

```diff
@@ -82,26 +80,18 @@ const verifySecret = async (id, secretKey) => {
  * @param {string} id
  * @returns {Promise<any>}
  */
-const findById = async id =>
-  Event.findById(id, { attributes: { exclude: ["secretKey"] } });
+const findById = async id => Event.findById(id, { secretKey: 0, __v: 0 });
```

Comment:
Same thing with the exec here. Also I wonder if we want to remove the `__v` here instead of when we're about to serve JSON as a response. If someone else uses `findByID` and then does an update on that event, mongo won't be able to use the document version to do the update and we might run into issues.